common/bls.py

```diff
@@ -65,8 +65,38 @@ async def gather_and_aggregate_signatures(
             )
         )
         for node_id in node_ids
-    ]
-    signatures: list[dict[str, Any] | None] = await asyncio.gather(*tasks)
+    }
+
+    aggregate_timeout = 10 # in seconds
```

Comment:
This should be read from config

Reply:
Resolved in this [commit](https://github.com/zellular-xyz/zsequencer/commit/c4b6c2088997246e75916014a9b12b18b28c0263)